client/cli/go/cmds/heketi_storage.go

```diff
@@ -282,7 +282,11 @@ func createHeketiCopyJob(volume *api.VolumeInfoResponse) *batch.Job {
 }
 
 var setupHeketiStorageCommand = &cobra.Command{
-	Use:   "setup-openshift-heketi-storage",
+	Use: "setup-openshift-heketi-storage",
+	Aliases: []string{
+		"setup-heketi-self-storage",
```

Comment:
I am thinking how about "setup-heketi-db-storage" instead of "setup-heketi-self-storage"? ...